src/screens/OrganizationPeople/OrganizationPeople.tsx

```diff
@@ -182,6 +182,17 @@ function OrganizationPeople(): JSX.Element {
                 onChange={debouncedHandleEventTitleSearchChange}
               />
               <div className={styles.radio_buttons}>
+                <input
+                  id="userslist"
+                  value="userslist"
+                  name="displaylist"
+                  type="radio"
+                  defaultChecked={state == 2 ? true : false}
+                  onClick={() => {
+                    setState(2);
+                  }}
+                />
+                <label htmlFor="userslist">{t('users')}</label>
```

Comment:
Why was this moved from from where it was?

Reply:
It was to make a proper order of : users < members < admins

Reply:
Okay got it 👍🏽